tools/python_tools/cuvslam_tools/tests/test_tum_preparation.py

```
@@ -0,0 +1,69 @@
+
+    def tearDown(self):
+        self._temporary.cleanup()
+
+    def _archive(self, build) -> Path:
+        archive = self.root / "archive.tgz"
+        with tarfile.open(archive, "w:gz") as tar:
+            build(tar)
+        return archive
+
+    def test_traversing_member_is_rejected(self):
+        def build(tar):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `build`

Add return type annotation: `None`

(ANN202)